src/index.ts

```diff
@@ -151,6 +217,47 @@ export class CheckoutInstance {
     }
   }
 
+  /**
+   * Creates popup modal in the DOM to host the Checkout iframe.
+   *
+   * If the Checkout becomes completed or if the user clicks cancelled, automatically closes the popup.
+   */
+  private createPopupDOMTree(iframe: HTMLIFrameElement, hostElement: HTMLDivElement): HTMLElement {
+    // Div Tree
+    const dialog = document.createElement('dialog')
+    const dialogBody = document.createElement('div')
+
+    // Style element
+    const style = document.createElement('style')
+    style.setAttribute('type', 'text/css')
+
+    // Using polyfill if necessary
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+    // @ts-ignore
+    if (typeof dialog.showModal !== 'function') {
+      style.appendChild(document.createTextNode(dialogCss))
+
+      dialogPolyfill.registerDialog(dialog)
+    }
+
+    // Apply style
+    style.appendChild(document.createTextNode(epcsdkCss))
+    document.head.appendChild(style)
+
+    // Set Attributes
+    dialog.setAttribute('id', `${hostElement.id}-modal`)
```

Comment:
We no longer need this id, correct? Instead it should have the class from the CSS above, where I suggested `.epcsdk-dialog` (but of course it can be something else).

Reply:
done.